src/Core/src/Handlers/ViewHandlerExtensions.Android.cs

```diff
@@ -132,5 +133,50 @@ internal static void PlatformArrangeHandler(this IViewHandler viewHandler, Rect
 
 			viewHandler.Invoke(nameof(IView.Frame), frame);
 		}
+
+		internal static void SetupContainerFromHandler(this IPlatformViewHandler viewHandler, Func<IPlatformViewHandler, View> createContainerView)
```

Comment:
Do these methods ever get called from anywhere other than ViewHandlerOfT.Android? Are there plans to call them from somewhere else in the future?

Reply:
:-/ Left over code from trying to fix `Frame` which I'll address [here](https://github.com/dotnet/maui/pull/10751). The point of this code will be to allow things that don't inherit from `ViewHandler` directly to reuse our `Container` concepts. This was the point where I realized that I needed to make two different PRs :-) All changes to the actual SDK should be removed from this PR now.